dev/com.ibm.ws.jaxws.ejb_fat/test-applications/EJBWSBasicClient/src/com/ibm/ws/jaxws/ejbbasic/client/EJBBasicClientServlet.java

```diff
@@ -57,6 +57,10 @@ public class EJBBasicClientServlet extends HttpServlet {
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
         String testMethod = req.getParameter("testMethod");
+
+        // Remove automatically added extension to match proper method name
+        testMethod = testMethod.replace("_EE10_FEATURES", "");
```

Comment:
Why isn't this needed for the EE9 repeat?

Reply:
_EE9_FEATURES string was removed in runTest method of EJBWSBasicTest class. I moved this removal to the same spot.